package.json

```diff
@@ -86,7 +86,7 @@
 		"jetpack-js-tools": "workspace:*"
 	},
 	"engines": {
-		"node": "^16.7.0",
+		"node": "14.17.6 || ^16.7.0",
```

Comment:
I think it is likely safe to allow any 14.x.x release. We, of course, would make this change to all `package.json` files in this PR. WDYT? 
```suggestion
		"node": "^14.17.6 || ^16.7.0",
```

Reply:
Thanks for bringing that up. I agree it's a good idea and have updated all the packages.